### PR TITLE
Update transfer related function signatures

### PR DESCRIPTION
**Pull request by weilbith**
Closes: #272 #273

### CODE REVIEW

File: tests/unit/Transaction.test.ts [hunk 1]
```diff
@@ -47,13 +47,7 @@ describe('unit', () => {
...
           'CurrencyNetwork',
           'transfer',
-          [
-            COUNTER_PARTY_ADDRESS,
-            '10000',
-            '0',
-            [COUNTER_PARTY_ADDRESS],
-            EXTRA_DATA
-          ]
+          ['10000', '0', [COUNTER_PARTY_ADDRESS], EXTRA_DATA]
```
Comment (berndbohmeier) [inline tests/unit/Transaction.test.ts]: the path should be [USER_ADDRESS, COUNTER_PART_ADDRESS]

File: tests/unit/Transaction.test.ts [hunk 2]
```diff
@@ -92,13 +86,7 @@ describe('unit', () => {
...
           'CurrencyNetwork',
           'transfer',
-          [
-            COUNTER_PARTY_ADDRESS,
-            '10000',
-            '0',
-            [COUNTER_PARTY_ADDRESS],
-            EXTRA_DATA
-          ],
+          ['10000', '0', [COUNTER_PARTY_ADDRESS], EXTRA_DATA],
```
Comment (berndbohmeier) [inline tests/unit/Transaction.test.ts]: same

File: tests/unit/Transaction.test.ts (hunk 1 above)
Comment (weilbith) [reply tests/unit/Transaction.test.ts]: :heavy_check_mark:

File: tests/unit/Transaction.test.ts (hunk 2 above)
Comment (weilbith) [reply tests/unit/Transaction.test.ts]: :heavy_check_mark:

**Pull request merged**